src/exposurelog/utils.py

```diff
@@ -0,0 +1,11 @@
+__all__ = ["current_date_and_day_obs"]
+
+import astropy.time
+import astropy.units
+
+
+def current_date_and_day_obs() -> tuple[astropy.time.Time, int]:
+    """Return the current date and day_obs."""
```

Comment:
Maybe add a note here saying that this matches the LSST day_obs definition of offsetting UTC by 12 hours?

Reply:
I sure hope that is TAI - 12 hours, but I'll ask on Slack.